File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
class Solution {
    public String solution(String s) {
        int min = Integer.MAX_VALUE;
        int max = Integer.MIN_VALUE;
        String arr[] = s.split(" ");
        for(int i=0; i<arr.length; i++){
            min = Math.min(min,Integer.parseInt(arr[i]));
            max = Math.max(max,Integer.parseInt(arr[i]));
        }
        String answer =  min + " " + max;
        return answer;
    }
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

string solution(string s) {
    vector<int> arr;
    string tmp = "";
    for(int i=0; i<=s.length(); i++){
        if(i<s.length() && s[i] != ' '){
            tmp += s[i];
        }
        else{
            if(!tmp.empty()){
                arr.push_back(stoi(tmp));
                tmp = "";
            }
        }
    }
    sort(arr.begin(), arr.end());
    string answer = to_string(arr[0]) + " " + to_string(arr[arr.size()-1]);
    return answer;
}